src/data/utils.js

```diff
@@ -1,10 +1,26 @@
 import User from '../users/user'
 
-export function resolveModelClassFromString (className) {
-  if(className === User.className){
+export function resolveModelClassFromString(className) {
+  if (className === User.className) {
     return User
   }
 
-  return function() {
+  return function () {
   }
 }
+
+export function convertObject(obj, result = {}) {
```

Comment:
I believe if you define a method `toJSON` in class `Geometry ` the util method will be unnecessary 

```
class Geometry {
...
   toJSON(){
      return this.asGeoJSON()  
   }
...
}
```

Reply:
Added in next commit